src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big"
+    class="formBig"
+    method="POST"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <img
+        class="bigMagnifyingGlass"
```

Comment:
Використовуй kebab-case, camelCase залишимо для JS-a :)
```suggestion
        class="big-magnifying-glass"
```